ta-sdk-spi/pom.xml

```diff
@@ -81,6 +81,11 @@
             <artifactId>commons-cli</artifactId>
             <version>1.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.leadpony.justify</groupId>
+            <artifactId>justify</artifactId>
+            <version>2.0.0</version>
```

Comment:
Please confirm license for this dependency is ok. Also open a corresponding issue in TA to get this dependency legal clearance